cats/cats.gradle

```diff
@@ -1,4 +1,4 @@
-tasks.bintrayUpload.enabled = false
+tasks.findByName('bintrayUpload')?.enabled = false
```

Comment:
There is probably a better way to accomplish this, I'm not entirely sure why we were explicitly trying not to publish this - probably because it is an intermediate nested build. Maybe worth just flattening out the directory structure...

Reply:
Yeah, I hate the way Gradle assumes any intermediate directory is its own sub-project.